README.md

```diff
@@ -22,33 +33,47 @@ git submodule update
+| `SEQUENCER`              | Address of [Sequencer](https://github.com/makerdao/dss-cron/)                                                                                                                                      |
+| `NETWORK_NAME`           | Short name from the Sequencer network registry                                                                                                                                                     |
+| `DSS_VEST`               | Address of [DssVest](https://github.com/makerdao/dss-vest)                                                                                                                                         |
+| `DAI_JOIN`               | Address of [DaiJoin](https://docs.makerdao.com/smart-contract-modules/collateral-module/join-detailed-documentation#3-key-mechanisms-and-concepts)                                                 |
+| `VOW`                    | Address of [Vow](https://docs.makerdao.com/smart-contract-modules/system-stabilizer-module/vow-detailed-documentation)                                                                             |
+| `PAYMENT_TOKEN`          | Address of ERC-20 token used for payment in DssVest, like DAI                                                                                                                                      |
+| `KEEPER_REGISTRY`        | Address of KeeperRegistry                                                                                                                                                                          |
+| `SWAP_ROUTER`            | Address of Uniswap V3 Router                                                                                                                                                                       |
+| `LINK_TOKEN`             | Address of ERC-20 token used for payment in KeeperRegistry, like LINK                                                                                                                              |
+| `PAYMENT_USD_PRICE_FEED` | Chainlink price feed for the DAI / USD pair or the associated `PAYMENT_TOKEN` and USD                                                                                                              |
+| `LINK_USD_PRICE_FEED`    | Chainlink price feed for the LINK / USD pair                                                                                                                                                       |
+| `MIN_WITHDRAW_AMT`       | Minimum amount of `PAYMENT_TOKOEN` required to trigger top up                                                                                                                                      |
+| `MAX_DEPOSIT_AMT`        | Maximum amount of `PAYMENT_TOKOEN` allowed to fund an upkeep. The excess amount is returned to the [Surplus Buffer](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer). |
```

Comment:
```suggestion
| `MIN_WITHDRAW_AMOUNT`       | Minimum amount of `PAYMENT_TOKOEN` required to trigger top up                                                                                                                                      |
| `MAX_DEPOSIT_AMOUNT`        | Maximum amount of `PAYMENT_TOKOEN` allowed to fund an upkeep. The excess amount is returned to the [Surplus Buffer](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer). |
```

Nit: Since all of the other env variables are named in full we can follow that convention here as well. Or is there another reasoning for the shortening?